.github/workflows/main.yml

```diff
@@ -24,3 +24,27 @@ jobs:
         uses: eskatos/gradle-command-action@v1
         with:
           arguments: :server:test -Dtests.crate.run-windows-incompatible=${{ matrix.os == 'ubuntu-latest' }}
+
+  linkcheck:
+    name: Sphinx linkcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Gradle Java
```

Comment:
do we need java, also probably the name `Setup Gradle Java` is not relevant here.

Reply:
Ah right. :facepalm: